wifiphisher/common/phishingpage.py

```diff
@@ -1,19 +1,14 @@
-"""
-This module handles all the phishing related operations for
-Wifiphisher.py
-"""
+"""This module handles all the phishing related operations."""
```

Comment:
Handles all the phishing related operations.

Reply:
A good practice to increase the code quality is to add extended documentation in the codebase.

You can take snippets from our documentation on readthedocs. For example, the following paragraph copied from the docs would fit here:

For specific target-oriented attacks, creating a custom Wifiphisher phishing scenario (or phishing template) may be necessary. For example, during a penetration testing, it may be necessary to capture the domain credentials using a phishing page with a familiar (to the victim users) interface, then verify the captured credentials over a local LDAP server and finally deliver them via SMTP to a mail server that we own."